docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
Minor, but maybe the `bat` file isn't necessary if we know we're building only on non-windows CI?

Reply:
You mean, no one will *bat* an eye if it is gone? :wink: